src/lib.rs

```diff
@@ -139,6 +139,23 @@ impl fmt::Display for ParseHermitVersionError {
 
 impl Error for ParseHermitVersionError {}
 
+/// A Uhyve interface version.
+#[derive(PartialOrd, Ord, PartialEq, Eq, Clone, Copy, Debug)]
+pub struct UhyveIfVersion {
```

Comment:
Not sure if we really need a newtype for this. An u32 is probably sufficient

Reply:
Yeah, could do. I mostly did that for the `impl TryFrom<Note<'_>> for UhyveIfVersion`, found it more convenient this way, as I wanted to avoid bloating `src/elf.rs` with more logic.